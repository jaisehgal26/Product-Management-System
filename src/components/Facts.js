import React from "react";
import "../App.css";
import CountUp from "react-countup";
import Fade from "react-reveal/Fade";
export default function Facts() {
  return (
    <div className="container-xxl py-6">
      <div className="container">
        <div className="row g-4">
          <Fade bottom>
            <div className="fact-card col-lg-3 col-md-6 ">
              <div className="fact-item rounded text-center h-100 p-5">
                <i className="fa fa-certificate fa-4x text-primary mb-4"></i>
                <p className="mb-2">Years Experience</p>
                <h1 className="display-5 mb-0" data-toggle="counter-up">
                  <CountUp end={100} duration={2.5} className="Playfair" />
                </h1>
              </div>
            </div>
          </Fade>
          <Fade bottom>
            <div className="fact-card col-lg-3 col-md-6 ">
              <div className="fact-item rounded text-center h-100 p-5">
                <i className="fa fa-users fa-4x text-primary mb-4"></i>
                <p className="mb-2">Skilled Professionals</p>
                <h1 className="display-5 mb-0" data-toggle="counter-up">
                  <CountUp end={175} duration={2.5} className="Playfair" />
                </h1>
              </div>
            </div>
          </Fade>
          <Fade bottom>
            <div className="fact-card col-lg-3 col-md-6 ">
              <div className="fact-item  rounded text-center h-100 p-5">
                <i className="fa fa-bread-slice fa-4x text-primary mb-4"></i>
                <p className="mb-2">Total Products</p>
                <br />
                <h1 className="display-5 mb-0" data-toggle="counter-up">
                  <CountUp end={135} duration={2.5} className="Playfair" />
                </h1>
              </div>
            </div>
          </Fade>
          <Fade bottom>
            <div className="fact-card col-lg-3 col-md-6 ">
              <div className="fact-item rounded text-center h-100 p-5">
                <i className="fa fa-cart-plus fa-4x text-primary mb-4"></i>
                <p className="mb-2">Order Everyday</p>
                <br />
                <h1 className="display-5 mb-0" data-toggle="counter-up">
                  <CountUp end={9357} duration={2.5} className="Playfair" />
                </h1>
              </div>
            </div>
          </Fade>
        </div>
      </div>
    </div>
  );
}
