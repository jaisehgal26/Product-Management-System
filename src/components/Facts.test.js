import * as React from "react";
import { shallow } from "enzyme";
import toJson from "enzyme-to-json";
import Facts from "./Facts";

test("should render correctly", () => {
  const wrapper = toJson(shallow(<Facts />));
  expect(wrapper).toMatchSnapshot();
});
