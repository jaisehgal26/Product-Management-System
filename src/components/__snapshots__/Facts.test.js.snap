// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div
  className="container-xxl py-6"
>
  <div
    className="container"
  >
    <div
      className="row g-4"
    >
      <Fade
        bottom={true}
      >
        <div
          className="fact-card col-lg-3 col-md-6 "
        >
          <div
            className="fact-item rounded text-center h-100 p-5"
          >
            <i
              className="fa fa-certificate fa-4x text-primary mb-4"
            />
            <p
              className="mb-2"
            >
              Years Experience
            </p>
            <h1
              className="display-5 mb-0"
              data-toggle="counter-up"
            >
              <CountUp
                className="Playfair"
                duration={2.5}
                end={100}
              />
            </h1>
          </div>
        </div>
      </Fade>
      <Fade
        bottom={true}
      >
        <div
          className="fact-card col-lg-3 col-md-6 "
        >
          <div
            className="fact-item rounded text-center h-100 p-5"
          >
            <i
              className="fa fa-users fa-4x text-primary mb-4"
            />
            <p
              className="mb-2"
            >
              Skilled Professionals
            </p>
            <h1
              className="display-5 mb-0"
              data-toggle="counter-up"
            >
              <CountUp
                className="Playfair"
                duration={2.5}
                end={175}
              />
            </h1>
          </div>
        </div>
      </Fade>
      <Fade
        bottom={true}
      >
        <div
          className="fact-card col-lg-3 col-md-6 "
        >
          <div
            className="fact-item  rounded text-center h-100 p-5"
          >
            <i
              className="fa fa-bread-slice fa-4x text-primary mb-4"
            />
            <p
              className="mb-2"
            >
              Total Products
            </p>
            <br />
            <h1
              className="display-5 mb-0"
              data-toggle="counter-up"
            >
              <CountUp
                className="Playfair"
                duration={2.5}
                end={135}
              />
            </h1>
          </div>
        </div>
      </Fade>
      <Fade
        bottom={true}
      >
        <div
          className="fact-card col-lg-3 col-md-6 "
        >
          <div
            className="fact-item rounded text-center h-100 p-5"
          >
            <i
              className="fa fa-cart-plus fa-4x text-primary mb-4"
            />
            <p
              className="mb-2"
            >
              Order Everyday
            </p>
            <br />
            <h1
              className="display-5 mb-0"
              data-toggle="counter-up"
            >
              <CountUp
                className="Playfair"
                duration={2.5}
                end={9357}
              />
            </h1>
          </div>
        </div>
      </Fade>
    </div>
  </div>
</div>
`;
